influxdb/influxdb_configuration.txt.tmpl

```diff
@@ -1,13 +0,0 @@
-# DDL
-CREATE DATABASE metrics
```

Comment:
where does this database live? Is it something we need to delete?

Reply:
The database lives inside influxcloud so it should just be a case of cancelling our description